api/auth_test.go

```diff
@@ -1,22 +0,0 @@
-package api
```

Comment:
Prefer to keep this test or something like that if possible.

Reply:
It just moved to `internal/tokens/tokens_test.go` when the helper methods moved.